faq.md

```diff
@@ -3,3 +3,27 @@ layout: page
 title: FAQ
 permalink: /faq
 ---
+# How can I help?
+
+If you have a service like this for people already, get in contact with us at [contact@vacfind.org][1]!
```

Comment:
I created a google form in #2  for submitting new sites or sending in corrections and encouraging people to cite sources. i wouldn't rely on email too much for stuff that could get very high volume

also, could we replace this with the jekyll veriable?